packages/strapi-plugin-users-permissions/config/queries/bookshelf.js

```diff
@@ -139,9 +139,7 @@ module.exports = {
 
   removePermission: async function (params) {
```

Comment:
This function doesn't look like it should be https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/config/queries/bookshelf.js#L140